addons/stock/report/report_stock_reception.py

```diff
@@ -50,21 +50,19 @@ def _get_report_values(self, docids, data=None):
             product_to_assigned_qty[assigned.product_id] += assigned.product_qty
 
         for move in move_ids:
+            move_quantity = move.product_qty or move.quantity
```

Comment:
Isn't it redundant? In the compute method for quantity, it is already computed with respect to the product uom

Reply:
Nope, it's all part of the fun of stock moves vs move lines :rainbow: For some reason we have a field that stores the ml's "quantity" in the product's uom, but for moves, we only store the "demand" in the product's uom and not the "quantity"

stock.move.quantity = quantity in move's uom (since this is what is displayed to the user, it should match the uom for the move)
stock.move.line.quantity_product_uom = quantity in product's uom (used for quant reserved/done quantity if I recall correctly)